doc/api/stream.md

```diff
@@ -3428,7 +3428,7 @@ added:
 * Returns: {integer}
 
 Returns the default highWaterMark used by streams.
-Defaults to `16384` (16 KiB), or `16` for `objectMode`.
+Defaults to `65536` (64 KiB), or `16384` (16 KiB) for `objectMode`.
```

Comment:
```suggestion
Defaults to `65536` (64 KiB), or `16` for `objectMode`.
```

this latest change is wrong. when in `objectMode` the `highWaterMark` is not measured in KB but by emitted items/objects